### ISSUE TITLE
Tensorflow with XLA hangs with both GPU and CPU at ~0% usage when training.

### ISSUE THREAD

**Issue by chasep255** [tensorflow/tensorflow#9509]
I am using the newest tensorflow which I built from source as of yesterday in an attempt to fix this issue.  Originally I had a source build of tensorflow 1.1.0.  I am running Ubuntu 16.04 with CUDA 8 and CUDNN 5.  My GPU is a GTX 1080.

The problem I am having is when I try to train my character based translator model using the XLA compiler.  The code makes it all the way through the initialize variables, etc up to the first run command which contains my train step and then just freezes.  Both my GPU and CPU are idle.  I attached gdb to my process and it seems to be stuck waiting for some sort of notification.  My model builds and runs fine if I am running it without training in predict mode but still with XLA.  It also runs fine if I train it without XLA.  Just the combo of XLA and training is the issue.

I attached to this my code plus some sample training data.  This problem should be reproducible by running the train.py.

[CharacterTranslator.zip](https://github.com/tensorflow/tensorflow/files/963946/CharacterTranslator.zip)

**Comment by chasep255** [comment]
Did you manage to make it anywhere debugging this?

**Comment by jiangxu87** [comment]
I have similar problem after compiling tf-1.1.0 with XLA support.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assigneee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tengyifei** [comment]
Similar problem with Tensorflow 1.5.0-rc0 with XLA support. Model trains just fine with XLA turned off but hangs with XLA on.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 134 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 149 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 164 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 179 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 194 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @hawkinsp: It has been 209 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by hawkinsp** [comment]
Closing issue as stale. Please reopen if you still experience this problem.
